doc/source/user/c-info.ufunc-tutorial.rst

```diff
@@ -157,42 +157,57 @@ the module.
             return m;
         }
 
-To use the ``setup.py`` file, place ``setup.py`` and ``spammodule.c``
-in the same folder. Then ``python setup.py build`` will build the module to
-import, or ``python setup.py install`` will install the module to your
-site-packages directory.
+.. note::
 
-    .. code-block:: python
+   For modern Python packaging, it is recommended to use a `pyproject.toml` and select
+   either `meson-python` or `scikit-build-core` as your build backend. Both are well-designed,
```

Comment:
For the above 2 lines: single backticks are references in ReST, can you use double backticks instead?